fastlane/Fastfile

```diff
@@ -413,16 +412,26 @@ lane :testflight_build do
     output_directory: "archives"
   )
 
-  changelog = "This is the official sample app built with Stream’s iOS Chat SDK v4.0. It’s designed to highlight engaging features and new improvements to the SDK, but remember that this is just one possible implementation. You can start your own by borrowing and customizing the code from this sample, or build something completely different using Stream’s components."
+  external_groups = []
+  changelog =
+    if ENV['GITHUB_EVENT_NAME'] == 'pull_request'
+      "This is the build for Regression testing on release candidate v#{version_number}"
+    else
```

Comment:
When do we get to `else` block?

Reply:
On any other `GITHUB_EVENT_NAME`, for example, `push`, `tag`, `release`, etc